src/utils/node_byte_utils.ts

```diff
@@ -23,6 +23,25 @@ type NodeJsBufferConstructor = Omit<Uint8ArrayConstructor, 'from'> & {
 // Node.js global
 declare const Buffer: NodeJsBufferConstructor;
 
+/** @internal */
+export function mathRandomBytes(byteLength: number) {
+  return nodeJsByteUtils.fromNumberArray(
+    Array.from({ length: byteLength }, () => Math.floor(Math.random() * 256))
+  );
+}
+
+/** @internal */
+const localRandomBytes = (() => {
+  try {
+    // What about nodejs es module users.........
```

Comment:
Have to discuss this, may change the goals of NODE-4860

Reply:
This is going to be a problem for the way we planned on bundling the library in NODE-4860, so it does not block this PR. I'll bring it up to the team when that ticket / PR is ready, I'll clean up my comment here for now, it just occurred to me when I wrote that line, didn't want to forget.